src/main/java/data/Athlete.java

```diff
@@ -0,0 +1,42 @@
+package data;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Created by p998tbd on 2016.12.15.
+ */
+public class Athlete implements  Comparable<Athlete>{
+    String name;
```

Comment:
If you do POJO, all friends should marked private and accessible only getters and/or setters